src/components/index-page/IndexOneSmallStep.tsx

```diff
@@ -99,15 +107,15 @@ export const IndexOneSmallStep = () => {
               {...register("projectDescription")}
             />
             <select
-              className={`index-one-small-step__select ${errors.inquiryLeadType ? "index-one-small-step__input__error" : ""}`}
+              className={`index-one-small-step__select ${
+                errors.inquiryLeadType ? "index-one-small-step__input__error" : ""
```

Comment:
in a prior PR I suggested making a helper function for this to eliminate duplication.